src/cc65/codeent.c

```diff
@@ -615,6 +615,9 @@ void CE_GenRegInfo (CodeEntry* E, RegContents* InputRegs)
         case OP65_BRK:
             break;
 
+        case OP65_STP:
+            break;
+
```

Comment:
Please move that case to a position in alphabetical order.  That is, place it just above `case OP65_STX:`.